app/src/tasks/TaskDetail/Metrics.tsx

```diff
@@ -0,0 +1,110 @@
+import { Await, useLoaderData } from "react-router-dom";
+import Col from "react-bootstrap/Col";
+import { Suspense } from "react";
+import { Aggregator, Task } from "../../ApiClient";
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+import { DateTime } from "luxon";
+import Placeholder from "react-bootstrap/Placeholder";
+import { OutLink } from "../../util";
+
+function FailedMetric({ name, counter }: { name: string; counter: number }) {
+  if (counter > 0) {
+    return (
+      <ListGroup.Item>
+        {name}: {counter}
+      </ListGroup.Item>
+    );
+  } else {
+    return null;
+  }
+}
+
+function UploadMetrics(task: Promise<Task>) {
+  return (
+    <Col md="6">
+      <Card className="my-3">
+        <Card.Body>
+          <Card.Title>Upload Metrics</Card.Title>
+          <Card.Subtitle>
+            <OutLink href="https://docs.divviup.org/product-documentation/operational-metrics#upload-metrics">
+              View Documentation
+            </OutLink>
+          </Card.Subtitle>
+        </Card.Body>
+        <Suspense fallback="0">
+          <Await resolve={task}>
+            {(task: Task) => (
+              <ListGroup variant="flush">
+                <ListGroup.Item>
+                  Successful Uploads: {task.report_counter_success}
```

Comment:
we may want to hide this whole card if the aggregator doesn't support upload metrics

Reply:
That's my intent here https://github.com/divviup/divviup-api/pull/809/files/679da779c03fb4a3f3e434400e12e4e1f493ab5a#diff-1e726048b6d1af4d0d2ba4695c7c012df0cb4da977923b071abc9ecfb30463a0R103-R105. Granted, I may have done this wrong (still iterating/testing).